packages/new-polymath-issuer/src/pages/DividendDetails/Presenter.tsx

```diff
@@ -465,9 +476,15 @@ export const Presenter = ({
           </CardPrimary>
         </GridRow.Col>
       </GridRow>
-      <Heading variant="h2" mt="l" mb="m">
-        Transactions
-      </Heading>
+      {loading ? (
+        <Box mt="l" mb="m">
+          <LoadingDots />
+        </Box>
+      ) : (
+        <Heading variant="h2" mt="l" mb="m">
+          Transactions
+        </Heading>
+      )}
```

Comment:
Why is there just the title being replaced by the loader? Shouldn't it be the entire table instead?

Reply:
I did the loading on the title to go with the cards at the top, but I can change that to replace the table with the loading indicator instead

Reply:
Not sure to understand. The title doesn't change/load anything. So I don't get why there should be a loading there at all! 😅

Reply:
I changed the loading indicator to replace the transactions table instead of the title.